scripts/azure_tags_env_variables.py

```diff
@@ -0,0 +1,16 @@
+from subprocess import Popen, PIPE
```

Comment:
Can we add a comment describing what this does?

Reply:
yep!